plugin/hover.py

```diff
@@ -274,10 +274,13 @@ def _show_hover(self, listener: AbstractViewListener, point: int, only_diagnosti
         if userprefs().show_symbol_action_links and contents and not only_diagnostics and hover_content:
             symbol_actions_content = self.symbol_actions_content(listener, point)
             if link_content and link_has_standard_tooltip:
-                symbol_actions_content += ' | ' + link_content
+                if symbol_actions_content:
+                    symbol_actions_content += ' | '
+                symbol_actions_content += link_content
             elif link_content:
                 contents += '<div class="link with-padding">' + link_content + '</div>'
-            contents += '<div class="actions">' + symbol_actions_content + '</div>'
+            if symbol_actions_content:
+                contents += '<div class="actions">' + symbol_actions_content + '</div>'
         elif link_content:
             contents += '<div class="{}">{}</div>'.format('link with-padding' if contents else 'link', link_content)
```

Comment:
Do you see a way to refactor this to not have 6 if else conditions here?

Reply:
Or at least not have 3 nested if conditions. 
To me it seems that there are many if statements. It would be good if there were less.